fun/meme.cc.lua

```diff
@@ -0,0 +1,37 @@
+{{/*
+	This command allows you to make a meme. Usage: `-meme <meme template> >top text> <bottom text> <link if template is custom>`.
+	You can provide a link for a custom meme, or use the existing templates: both, buzz, doge, joker and sad-biden.
+	Sometimes, the special characters will mess up the link, so it is recommended that you only use these special characters: " " "?" "%" "#" "/"
+  Example: `-meme custom "this is" "my meme generator" "https://wiki.geogebra.org/uploads/a/a9/Example.jpg"`
+	Recommend trigger: Command trigger with trigger `meme`. If you have any questions, contact me on discord: GenryMg#0001
+*/}}
+
+
+{{$link := "https://memegen.link"}}
+{{$args := parseArgs 3 (`Usage: -meme <meme template> "top text" "bottom text"`)
+(carg "string" "User")
```

Comment:
should have some better carg descriptions here

Reply:
yeah... those were copy and pasted, i forgot to change it